workspaces/ui-v2/src/pages/diffs/ReviewDiffPages.tsx

```diff
@@ -35,6 +35,10 @@ export function DiffReviewPages(props: any) {
   const allRequestsAndResponsesOfBaseSpec = useAllRequestsAndResponses();
   useFetchEndpoints();
   const endpointsState = useAppSelector((state) => state.endpoints.results);
+  const filteredEndpoints = useMemo(
```

Comment:
We need to useMemo this, since changes in identity here cause something in the SharedDiffStore to reset state. This should be less brittle when we move it to redux, just wanted to call it out since it's easy to make a mistake here and break the diff flow.